third_party/terraform/utils/utils.go.erb

```diff
@@ -490,3 +490,54 @@ func getInterconnectAttachmentLink(config *Config, project, region, ic string) (
 
 	return ic, nil
 }
+
+type kmsKeyRingId struct {
```

Comment:
I'm fine with this change, though I'd prefer it in a kmsutils.go file or something similar